apis/app/definition.yaml

```
@@ -0,0 +1,68 @@
+apiVersion: apiextensions.crossplane.io/v1
+kind: CompositeResourceDefinition
+metadata:
+  name: xapps.aws.platformref.upbound.io
+spec:
+  defaultCompositeDeletePolicy: Foreground
+  group: aws.platformref.upbound.io
+  names:
+    kind: XApp
+    plural: xapps
+  claimNames:
+    kind: App
+    plural: apps
+  versions:
+  - name: v1alpha1
+    served: true
+    referenceable: true
+    schema:
+      openAPIV3Schema:
+        type: object
+        properties:
+          spec:
+            type: object
+            properties:
+              parameters:
+                type: object
+                description: App configuration parameters.
+                properties:
+                  deletionPolicy:
+                    description: When the Composition is deleted, delete the AWS resources. Defaults to Delete
```

```suggestion
                    description: Delete the external resources when the Claim/XR is deleted. Defaults to Delete
```
Nit: The removal of Claim/XR is the trigger for MR deletion